frontends/web/src/routes/account/walletconnect/components/header/header.module.css

```diff
@@ -1,7 +1,8 @@
 .headerContainer {
+    align-items: center;
     display: flex;
     flex-direction: column;
-    align-items: center;
+    margin-bottom: var(--space-half);
```

Comment:
this is only needed on WC on mobile, when scanning or on success view. I'll move this further down inside the media query, so it only applies on small screen.